src/html-templates.ts

```diff
@@ -9,10 +9,17 @@ import {
   getCssProperties,
   getSlots,
 } from "./cem-utilities.js";
+import { Options } from "./storybook";
 
 let argObserver: MutationObserver | undefined;
 let lastTagName: string | undefined;
 
+let options: Options = {};
+
+setTimeout(() => {
```

Comment:
Did you need a timeout here?

Reply:
It seems like it, options don't get picked up otherwise. I copied this pattern from cem-utilities.ts.

Reply:
Ah yeah, that's right. It needs to wait for storybook to catch up.